service/history/timerQueueStandbyProcessor.go

```diff
@@ -124,6 +124,10 @@ func (t *timerQueueStandbyProcessorImpl) process(timerTask *persistence.TimerTas
 		scope = metrics.TimerTaskWorkflowTimeoutScope
 		err = t.processWorkflowTimeout(timerTask)
 
+	case persistence.TaskTypeRetryTimer:
+		scope = metrics.TimerTaskRetryTimerScope
+		err = t.processRetryTimer(timerTask)
```

Comment:
the retry timer task, should not be processed by the standby timer processor, so this should be
```
scope = metrics.TimerTaskRetryTimerScope
err = nil
```